### DISCUSSION TITLE
Is there a way to create a VM on my Windows PC and then transfer (or convert) that VM to the iOS version of UTM SE?

### DISCUSSION THREAD

**Discussion by Banjo-Oz** [utmapp/UTM#6477] [UTM Help (iOS)]
I'd be much more comfortable making a VM image on my desktop PC, but would like to run it in UTM SE on iOS. Is this possible? I notice that UTM uses *.utm files but is this just a "friendly" extension name that is another format in reality, or a proprietary VM format?

Happy if this means having to install something else (QEMU, etc.) on my PC to do this.

**Comment by 100tomer** [comment]
+1
